cask.el

```diff
@@ -730,7 +730,7 @@ If DEEP is t, all dependencies recursively will be returned."
            (-map
             (lambda (dependency)
               (list (cask-dependency-name dependency)
-                    (cask-dependency-version dependency)))
+                    (or (cask-dependency-version dependency) "0")))
```

Comment:
I'm not sure whether I like this.  Generally, you should explicitly specify the minimum version for any runtime dependency.  

I know that we currently don't enforce this in Cask, but that's mostly to avoid inconveniences for those who use Cask for their Emacs configuration.  

@rejeep What do you think?